Sources/GRPCInteropTests/InteroperabilityTestCases.swift

```diff
@@ -15,6 +15,7 @@
  */
 
 internal import GRPCCore
+import SwiftProtobuf
```

Comment:
Presumably this should be `private`

```suggestion
private import SwiftProtobuf
```